kube-core/src/lib.rs

```diff
@@ -70,3 +70,6 @@ pub use error::ErrorResponse;
 
 mod version;
 pub use version::Version;
+
+pub mod error_boundary;
```

Comment:
for now since there's only one thing and it's top level exported
```suggestion
mod error_boundary;
```

Reply:
That would leave the error type with no pronouncable path.

Reply:
Ah ok, my bad.